### PR TITLE
fix something

### PR DESCRIPTION
**Pull request by EarthmanT**

### CODE REVIEW

File: ecosystem_cicd_tools/packaging.py [hunk 1]
```diff
@@ -194,24 +195,30 @@ def update_assets_in_plugin_dict(plugin_dict, assets, plugin_version=None):
...
-            elif wagon['name'] == CENTOS and 'centos-Core' in asset:
-                if asset.endswith('md5'):
-                        wagon['md5url'] = asset
-                else:
-                    wagon['url'] = asset
+        elif 'centos-altarch' in asset:
+            if asset.endswith('md5'):
+                wagons_list_copy[0]['md5url'] = asset
+            else:
+                wagons_list_copy[0]['md5url'] = asset
```
Comment (glukhman) [inline ecosystem_cicd_tools/packaging.py:210]: `wagons_list_copy[0]['url'] = asset` rather.
same for the others

File: ecosystem_cicd_tools/packaging.py [hunk 2]
```diff
@@ -194,24 +195,30 @@ def update_assets_in_plugin_dict(plugin_dict, assets, plugin_version=None):
...
-                if asset.endswith('md5'):
-                        wagon['md5url'] = asset
-                else:
-                    wagon['url'] = asset
-            elif wagon['name'] == CENTOS and 'centos-Core' in asset:
-                if asset.endswith('md5'):
-                        wagon['md5url'] = asset
-                else:
-                    wagon['url'] = asset
+        elif 'centos-altarch' in asset:
```
Comment (glukhman) [inline ecosystem_cicd_tools/packaging.py:206]: Can't we do something more future-proof here so if we later add more architectures (e.g. manylinux) we won't be tied to fixed indexes in the list?

File: ecosystem_cicd_tools/resources/plugins.json [hunk 3]
```diff
@@ -3,17 +3,17 @@
         "description": "A Cloudify Plugin that provisions resources in AWS",
         "releases": "https://github.com/cloudify-cosmo/cloudify-aws-plugin/releases",
         "title": "AWS",
-        "version": "2.12.10",
-        "link": "http://repository.cloudifysource.org/cloudify/wagons/cloudify-aws-plugin/2.12.10/plugin.yaml",
+        "version": "2.12.11",
+        "link": "http://repository.cloudifysource.org/cloudify/wagons/cloudify-aws-plugin/2.12.11/plugin.yaml",
         "wagons": [
```
Comment (glukhman) [inline ecosystem_cicd_tools/resources/plugins.json:8]: Altarch wagons? same for the others

File: ecosystem_cicd_tools/packaging.py (hunk 1 above)
Comment (EarthmanT) [reply ecosystem_cicd_tools/packaging.py]: durn good pt

File: ecosystem_cicd_tools/resources/plugins.json (hunk 3 above)
Comment (EarthmanT) [reply ecosystem_cicd_tools/resources/plugins.json:8]: This file I just keep everytime i make an update it is not actually used anywhere.

File: ecosystem_cicd_tools/packaging.py (hunk 2 above)
Comment (EarthmanT) [reply ecosystem_cicd_tools/packaging.py:206]: I think not. ATM wagons take the name that we give them in the wagon command. Wagon names are based largely on convention. I can change the name of a wagon just by giving it a different name in a command. So, we can say we are doing "future proofing", but unless we future proof wagon, it is for naught. Also, I highly doubt we are anywhere near manylinux support. Lol.

**Review by glukhman — APPROVED** [review]

**Pull request merged**